### PR TITLE
Fixed org teams only requirement in operator-connect-configure.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7349.

@cazawideh @CarolynRowe could you kindly help me here to confirm this information?

I also think that the 3rd bullet is wrong because of the following important note:
> [!IMPORTANT]
> **Emergency addresses:** Emergency addresses associated with a number acquired from the operator are managed directly with the operator. Please contact the operator to make any changes.

Thank you!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/operator-connect-configure.md [hunk 1]
```diff
@@ -105,10 +101,11 @@ To acquire numbers for new Teams users, follow these steps:
 
 4. Once your operator completes the order, they'll upload numbers to your tenant. You can view the numbers and the provider in the Teams admin center by going to **Voice > Phone numbers**. Assign Operator Connect numbers to users by using the Teams admin center or by using  PowerShell. For more information, see [Assign numbers](#assign-numbers).
 
-   
-
 ### Assign numbers
 
+> [!NOTE]
+> Before assign a number, make sure the user is in TeamsOnly mode. To check, see [Set upgrade options for a single user in your organization](setting-your-coexistence-and-upgrade-settings.md#set-upgrade-options-for-a-single-user-in-your-organization).
```
Comment (JohanFreelancer9) [inline Teams/operator-connect-configure.md:107]:
```suggestion
> Before assigning a number, make sure the user is in TeamsOnly mode. To check, see [Set upgrade options for a single user in your organization](setting-your-coexistence-and-upgrade-settings.md#set-upgrade-options-for-a-single-user-in-your-organization).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cazawideh PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cazawideh** [comment]
Looping in @livaht to confirm whether this is correct.

**Comment by cazawideh** [comment]
resolving in #7395

**Pull request closed without merging**